src/test/java/org/csanchez/jenkins/plugins/kubernetes/pipeline/steps/AssertBuildLogMessage.java

```diff
@@ -0,0 +1,24 @@
+package org.csanchez.jenkins.plugins.kubernetes.pipeline.steps;
+
+import org.jenkinsci.plugins.workflow.job.WorkflowJob;
+import org.jenkinsci.plugins.workflow.job.WorkflowRun;
+import org.jvnet.hudson.test.JenkinsRule;
+import org.jvnet.hudson.test.RealJenkinsRule;
+
+public class AssertBuildLogMessage implements RealJenkinsRule.Step {
```

Comment:
BTW you can also make these things `static` methods in some util class; a bit less boilerplate.

Reply:
Yeah, but less reusable then...

Reply:
How is that less reusable?

Reply:
When writing a new test it seems natural to go to the `steps` package and look for generic steps to use in the new test, however inspecting other test classes looking for static methods seems less discoverable to me at least. That will likely end up with re-writing the same (or very similar) static methods in multiple test classes.

Reply:
> inspecting other test classes looking for static methods

I meant reusable static methods in a utility class in the `steps` package.